mozilla-release/toolkit/mozapps/extensions/content/extensions.js

```diff
@@ -2765,6 +2770,65 @@ var gDetailView = {
     document.getElementById("detail-icon").src = icon ? icon : "";
     document.getElementById("detail-creator").setCreator(aAddon.creator, aAddon.homepageURL);
 
+    //CLIQZ-SPECIAL: This adds permission area to addon-details
```

Comment:
@tsl143 Would you like to put your peace of code into a separate function?
You could 've probably called it renderPermList or similar...

Reply:
I just feel like going with the flow of the function, its having all the view updates/render